plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/VidarrProvenanceDefinition.java

```diff
@@ -0,0 +1,10 @@
+package ca.on.oicr.gsi.shesmu.vidarr;
+
+import ca.on.oicr.gsi.shesmu.plugin.input.InputFormat;
+
+public class VidarrProvenanceDefinition extends InputFormat {
+
+  public VidarrProvenanceDefinition() {
+    super("vidarr_analysis", VidarrProvenanceValue.class);
```

Comment:
The input format is called 'vidarr_analysis' in shesmu by this line. Did you want to call it 'vidarr_provenance'?

Reply:
The other input format definition classes end with `XYFormatDefinition` . I do agree that it's unclear to have the class be Vidarr Provenance and the input format name be vidarr analysis; perhaps vidarr analysis for both? (so class becomes `VidarrAnalysisFormatDefinition`)